client/pages/upload/index.vue

```diff
@@ -61,9 +75,16 @@
       </widgets-alert>
 
       <!-- Item Upload cards -->
-      <template v-for="item in items">
-        <cards-item-upload-card :ref="`itemCard-${item.index}`" :key="item.index" :media-type="selectedLibraryMediaType" :item="item" :processing="processing" @remove="removeItem(item)" />
-      </template>
+      <cards-item-upload-card 
```

Comment:
For viz: minor refactor here to put the `:key` on the iterator. But you can't use a `template` element once you do that, so I removed it. Should be effectively no change